addons/event_crm/tests/test_event_crm_flow.py

```diff
@@ -166,3 +168,35 @@ def test_event_crm_trigger_done(self):
         registration.action_set_done()
 
         self.assertLeadConvertion(self.test_rule_order_done, registration)
+
+    def test_event_crm_lead_merge(self):
```

Comment:
Would move to a test_crm_lead_merge file (similar to the structure in standard CRM)